src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {

```

Comment:
make method non static

it is required by task
```suggestion
    public Ball getRandomBall() {

```